src/libraries/System.Security.Cryptography.Cng/tests/PropertyTests.cs

```diff
@@ -160,5 +160,31 @@ public static void NullValueRoundtrip()
             CngProperty property = new CngProperty("banana", null, CngPropertyOptions.CustomProperty);
             Assert.Null(property.GetValue());
         }
+
+        [Fact]
+        public static void TestConstructorSpan()
+        {
+            var name = "dotnet-test";
+            ReadOnlySpan<byte> value = [1, 2, 3, 4, 5];
+
+            CngProperty property = new CngProperty(name, value, CngPropertyOptions.CustomProperty);
+            AssertExtensions.SequenceEqual(value, property.GetValue());
+        }
+
+        [Fact]
+        public static void TestConstructorSpan_NameNull()
+        {
+            var name = "dotnet-test";
```

Comment:
The type name of a variable declaration has to always appear.  While string literals used to be obvious-typed (but still failed the var rule), with UTF-8 string literals it now requires reading the whole line and paying attention to see if something is string or ReadOnlySpan-byte... so now string literals are super-not-approved-for-var.

```suggestion
            string name = "dotnet-test";
```